Lombiq.Hosting.Tenants.Management/Constants/FeatureNames.cs

```diff
@@ -0,0 +1,10 @@
+namespace Lombiq.Hosting.Tenants.Management.Constants
+{
+    public static class FeatureNames
+    {
+        public const string Module = "Lombiq.Hosting.Tenants.Management";
+
+        public const string ForbindenTenantNames = Module + "." + nameof(ForbindenTenantNames);
```

Comment:
Ez a typo több helyen is előjön.

```suggestion
        public const string ForbiddenTenantNames = Module + "." + nameof(ForbiddenTenantNames);
```